api/core/model_runtime/model_providers/openai_api_compatible/llm/llm.py

```diff
@@ -343,22 +343,31 @@ def create_final_llm_result_chunk(index: int, message: AssistantPromptMessage, f
                 )
             )
 
-        for chunk in response.iter_lines(decode_unicode=True, delimiter='\n\n'):
+        # delimiter for stream response, need unicode_escape
+        import codecs
+        delimiter = credentials["stream_mode_delimiter"]
+        delimiter = codecs.decode(delimiter, "unicode_escape")
```

Comment:
# the encrypted_config encoded the stream_mode_delimiter, so need decode
{
    "api_key": "xxx",
    "endpoint_url": "http://192.168.68.100:18000/v1/",
    "mode": "chat",
    "context_size": "2000",
    "max_tokens_to_sample": "2100",
    "stream_mode_delimiter": "\\r\\n\\r\\n",
    "stream_mode_exclude_chunks": "[DONE]"
}